app/controllers/file_controller.rb

```diff
@@ -50,7 +50,9 @@ def show
     disposition = params[:disposition] || 'inline'
     filename = params["filename"] || "#{objid}#{fileid}"
     headers['Content-Disposition'] = "#{disposition}; filename=#{filename}"
-    if ds.mimeType
+    if filename.include?('.jpg')
```

Comment:
@lsitu The example file in the ticket bb0478952m_2.jpg, It display fine that doesn't get corrupted.  I can open it without any problem. Do you get any problem for it?

The ticket require to change Content-Type from application/octet-stream to image/jpeg, which we need to fix here:
https://github.com/ucsdlib/damspas/blob/develop/app/controllers/file_controller.rb#L58